### DISCUSSION TITLE
Can the capture range be set smaller?

### DISCUSSION THREAD

**Discussion by more77tai** [Nain57/Smart-AutoClicker#137] [Q&A]
Can the horizontal capture range be set smaller when capturing in landscape on a tablet?

**Comment by jmhjr** [comment]
You can zoom in on the area using pinch.

**Reply by more77tai** [reply]
Can the minimum interception range be set smaller
![This is an image}A](https://user-images.githubusercontent.com/74169990/170889309-9e11ac59-068b-4aaa-88de-a1f7dd7e7bfb.png)

**Reply by jmhjr** [reply]
You can adjust the section picture to be any size by pinching or zooming the screen.

**Reply by more77tai** [reply]
thanks, i found it
